docs/guides/segment-migration-guide.md

```diff
@@ -153,8 +153,8 @@ Both Segment and Amplitude SDKs are meant to capture first party data by trackin
     ```swift
     Amplitude.instance().setUserId("abc")
     let identify = AMPIdentify()
-        .set("email", value: "female")
-        .set("age",value: NSNumber(value: 20))
+      .set("email", value: "female")
+      .set("age",value: NSNumber(value: 20))
     Amplitude.instance().identify(identify)
```

Comment:
What about making this all one line? 

```
    Amplitude.instance().identify(AMPIdentify()
      .set("email", value: "female")
      .set("age",value: NSNumber(value: 20))
    )
```

Reply:
@justin-fiedler I noticed our SDKs don't follow the same format for identify. Should we update them too?


- [Browser](https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/#identifyset) 
- [iOS](https://www.docs.developers.amplitude.com/data/sdks/ios-swift/#identify)

Reply:
Good question @jooohhn , let me ask the team if the verbose usage was intentional there. We can update in another PR.